__tests__/app.test.js

```diff
@@ -84,6 +84,47 @@ describe("GET /api", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("STATUS 200: responds with a correct article object", () => {
+    const expectedOutput = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toEqual(expectedOutput);
```

Comment:
We need to be careful when we are asserting the structure of an object like this. What if we were to decide that we wanted to add a property to our data? We would have to amend this test for every property that we wanted to add if we are going along the line of checking the complete shape of the object. 

We can however target properties that we are expecting to be on that object and specifically assert that they will have a specific value.

Reply:
done! thank you!